group_vars/bibdata/production.yml

```diff
@@ -162,7 +162,7 @@ samba_shares:
   - name: marc_liberation_files/campus_access_files
     group: sambashare
     write_list: +sambashare
-    writable: yes
+    writable: true
```

Comment:
I wonder if we need to add `owner` here.  I do not understand looking at the code how deploy became the owner and the group of the directory.

Reply:
this will be done on Maintenance Monday with a different still issued PR.